ts/src/kucoin.ts

```diff
@@ -652,6 +652,9 @@ export default class kucoin extends Exchange {
                 'fetchMarkets': {
                     'fetchTickersFees': true,
                 },
+                'fetchOrderBook': {
+                    'spotEndpoint': undefined, // string "level2", "level2_20" or "level2_100" (by default, "level2_100"is used)
```

Comment:
this key is ambiguous, when we have "endpoint" it means literally the entire endpoint name, here I would call it `level` for example

Reply:
to look at it from another angle, the `endpoint` might be more intuitive than `level`, bcz users need at first to find out what is level (either unified or exchange specific, becuase it's not 100% clear what is level), and look into code to find out what it does actually.
while `endpoint` directly clarifies that it's exchange-specific endpoint which user can choose. maybe we can have eg `orderbook/level2_20` (similar approach like this https://github.com/ccxt/ccxt/blob/master/ts/src/pro/kucoin.ts#L706 ) ?

Reply:
@carlosmiei I've simplified the PR and now it's at its best version